metadata-ingestion/tests/integration/sql_server/golden_files/golden_mces_mssql_no_db_to_file.json

```diff
@@ -1255,8 +1422,73 @@
     "changeType": "UPSERT",
     "aspectName": "subTypes",
     "aspect": {
-        "value": "{\"typeNames\": [\"table\"]}",
-        "contentType": "application/json"
+        "json": {
+            "typeNames": [
+                "table"
+            ]
+        }
+    },
+    "systemMetadata": {
+        "lastObserved": 1615443388097,
+        "runId": "mssql-test"
+    }
+},
+{
+    "entityType": "dataFlow",
+    "entityUrn": "urn:li:dataFlow:(mssql,demodata.Foo.stored_procedures,PROD/localhost/demodata)",
```

Comment:
After URN changes, I was actually hoping for this to look like:

"urn:li:dataFlow:(mssql,localhost.demodata.Foo.stored_procedures,PROD"

in sync with how we construct dataFlow URNs in other sources.